### PR TITLE
feat: upgrade all

### LINKED ISSUE #10: Add Mastodon to social media links
Add a link to the mastodon account, like:
`<a rel="me" href="https://tooting.ch/@graasp">Mastodon</a>`

Use the `rel="me"` to enable the mastodon instance to verify the authenticity of the account.

### PR DESCRIPTION
**Pull request by spaenleh**
This PR:
- fix #10 
- upgrade dependencies
- update logos
- add deployment workflow

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+# yarn pretty-quick --staged && yarn lint # TODO: reconfigure
+yarn prettier --check "src/**/*.js" && yarn lint
```
Comment (spaenleh) [inline .husky/pre-commit]: change this to use the check command

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "prettier.prettierPath": ""
```
Comment (pyphilia) [inline .vscode/settings.json]: gitignore.

File: src/components/sections/Header.tsx [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+import GraaspLogo from '@images/logos/graasp.svg';
 import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
 
 const Header = () => (
   <HeaderWrapper>
     <Container>
       <Grid>
-        <Art>{GraaspLogo()}</Art>
+        <Art>{/* <GraaspLogo /> */}</Art>
```
Comment (pyphilia) [inline src/components/sections/Header.tsx]: Remove.

File: src/images/logos/graasp.svg
Comment (pyphilia) [inline src/images/logos/graasp.svg:1]: Let's use the new logo?

**Review by pyphilia — APPROVED** [review]

File: src/images/logos/graasp.svg
Comment (spaenleh) [reply src/images/logos/graasp.svg:1]: yes !

**Pull request merged**